.github/workflows/check.yaml

```
@@ -178,6 +178,60 @@ jobs:
+      - name: Dry-Run Upload (to get url)
+        id: preupload
+        uses: ably/sdk-upload-action@v2
+        with:
+          mode: preempt
+          sourcePath: .build/plugins/Swift-DocC/outputs/AblyChat.doccarchive # Path to the Swift DocC output folder
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          artifactName: AblyChat
+
+      # Build the documentation using Swift DocC
+      - name: Build documentation
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:1:1: PR_NUMBER appears unused. Verify use (or export if used externally)

(shellcheck)